### PR TITLE
Remove erroneous whitespace from CardinalSuffix return value

### PR DESCRIPTION
**Pull request by dbenage-cx**
PR addresses root issue mentioned in #1462 of asteroids having an extra space when not suffixed with a numeral.

### CODE REVIEW

**Comment by geoffthemedio** [comment]
@adrian_broher the travis build failure might warrant your attention.

**Comment by dbenage-cx** [comment]
rebased to master, looks like linux build stalled

**Comment by Vezzra** [comment]
Um, the Travis CI build failure was most likely my fault, I tinkered with the Xcode project. I've already reversed the offending commit.

File: universe/Planet.cpp [hunk 1]
```diff
@@ -481,50 +481,69 @@ float Planet::NextTurnCurrentMeterValue(MeterType type) const {
...
     if (Type() == PT_ASTEROIDS)
-        retval.append(UserString("NEW_ASTEROIDS_SUFFIX") + " ");
+        retval.append(UserString("NEW_ASTEROIDS_SUFFIX"));
 
     std::shared_ptr<System> cur_system = GetSystem(SystemID());
     if (cur_system) {
         if (cur_system->OrbitOfPlanet(ID()) < 0) {
             ErrorLogger() << "Planet " << Name() << "(" << ID() << ") "
                           << "has no current orbit";
+            if (Type() == PT_ASTEROIDS)
```
Comment (geoffthemedio) [inline universe/Planet.cpp]: why only for asteroids?

Comment (dbenage-cx) [reply universe/Planet.cpp]: The asteroids had a prefix applied and needed a space to separate the ordinal.
Determining to add a comment or change the functionality, I opted for later which cascaded into improving the flow of this function.

**Pull request merged**